app/commands/solution/search_user_solutions.rb

```diff
@@ -70,6 +76,9 @@ def search_query
             track_slug.blank? ? nil : { terms: { 'track.slug': [track_slug].flatten } },
             status.blank? ? nil : { terms: { status: [status].flatten } },
             mentoring_status.blank? ? nil : { terms: { mentoring_status: [mentoring_status].flatten } },
+            sync_status.nil? ? nil : { term: { 'out_of_date': sync_status == :out_of_date } },
+            tests_status.blank? ? nil : { terms: { 'published_iteration.tests_status': to_terms(tests_status) } },
+            head_tests_status.blank? ? nil : { terms: { 'published_iteration.head_tests_status': to_terms(head_tests_status) } },
```

Comment:
I currently have copied what we use for the searching of the community solutions. However, there we were only showing published solutions. Do we want to use the published iteration on the journey solutions page too? AFAICT, the tests status wasn't actually shown on the journey page, so this might not be bad.

Reply:
Good question. I think we should show the double tick on the card if it's got an iteration solution and passing HEAD and the general tests-status. With regards to this specific bit, I think what you have here is fine.